### PR TITLE
Create project and add users functionality

### PR DESCRIPTION
**Pull request by elcorbs**
screen shot 2019-02-15 at 12 32 06

### CODE REVIEW

File: src/Components/ProjectList/index.js [hunk 1]
```diff
@@ -38,6 +39,7 @@ export default class ProjectList extends React.Component {
 
   renderTitles = () => {
     return <tr>
+        <th>#</th>
```
Comment (ccp92) [inline src/Components/ProjectList/index.js]:
```suggestion
        <th>ID</th>
```
As we mention adding a user to a certain project ID, makes it a bit clearer if we label the number as ID here as well.

File: src/Components/AdminPortal/index.js [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+          
+        </select>
+      </div>
+    </div>
+  }
+
+  renderUserDetails = () => {
+    return <div>
+      <div className="form-group">
+        <label htmlFor="userEmail">Enter the users email here.</label>
```
Comment (ccp92) [inline src/Components/AdminPortal/index.js]:
```suggestion
        <label htmlFor="userEmail">Enter the user's email here.</label>
```
Not sure if the apostrophe needs to be escaped as it's in a tag?

File: src/Components/AdminPortal/index.js [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+        <input 
+          className="form-control"
+          data-test="user-email"
+          id="userEmail"
+          placeholder="User Email"
+          onChange={e => this.onFieldChange(e.target.value, "email")}
+        />
+      </div>
+      <div className="form-group">
+        <label htmlFor="userRole"> Please select the users role priveledges.</label>
```
Comment (ccp92) [inline src/Components/AdminPortal/index.js]:
```suggestion
        <label htmlFor="userRole"> Please select the user's role privileges.</label>
```

File: src/Components/AdminPortal/index.js [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+      </div>
+    </div>
+  }
+
+  renderAddUsers = () => {
+    return <div>
+        <h3>
+          Add a user to a project
+        </h3>
+        {this.renderAddUsersForm()}
```
Comment (ccp92) [inline src/Components/AdminPortal/index.js]: Might be worth adding a message here saying that you must complete the project creation process before attempting to add users?
I can see that people might fill in the new project section and add new user section at the same time, thinking it's one process.

File: src/Components/AdminPortal/index.js [hunk 5]
```diff
@@ -0,0 +1,216 @@
...
+        <div className="radio">
+          <label>
+            <input
+              type="radio"
+              data-test="user-role-su"
+              value="Superuser"
+              name="userRole"
+              onChange={e => this.onFieldChange(e.target.value, "role")}
+            />Superuser
+          </label>
```
Comment (ccp92) [inline src/Components/AdminPortal/index.js]: Just thinking. Do we not need an option for s151 officer?

File: src/Components/AdminPortal/index.js (hunk 4 above)
Comment (elcorbs) [reply src/Components/AdminPortal/index.js]: Theres a section in the project creation box to add a user though

File: src/Components/AdminPortal/index.js (hunk 5 above)
Comment (elcorbs) [reply src/Components/AdminPortal/index.js]: Good point, I've added this now

**Pull request merged**